.github/workflows/fedora_36.yml

```diff
@@ -0,0 +1,74 @@
+name: fedora_36
```

Comment:
> NO_DOC=ci
> NO_TEST=ci
> NO_CHANGELOG=ci

We announce support of new distros in release notes in the past. It is the user visible thing.

Reply:
fixed